core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/NameRepairStrategy.kt

```diff
@@ -0,0 +1,16 @@
+package org.jetbrains.kotlinx.dataframe.io
+
+/**
+ * This strategy defines how the repeatable name column will be handled
+ * during the creation new dataframe from the IO sources.
+ */
+public enum class NameRepairStrategy {
+    /** No actions, keep as is. */
+    NO,
```

Comment:
`DO_NOTHING`? Also, I wonder when this option might be used, since DataFrames cannot contain duplicate column names.